server/controllers/users.js

```diff
@@ -0,0 +1,198 @@
+import jwt from 'jsonwebtoken';
+import db from '../models';
+import Helpers from './helpers';
+
+/**
+ * Secret token for jsonwebtoken
+ */
+const secret = process.env.SECRET || 'my secret key';
+
+const usersController = {
+  /**
+   * Login a user
+   * @param {Object} req - Request object
+   * @param {Object} res - Response object
+   * @returns {Object} Response object
+   */
+  login(req, res) {
```

Comment:
Great! I understand your point. I'll work around it to ensure they both run at the same time. Thanks!